arch/arm64/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi

```diff
@@ -114,6 +114,10 @@ rp1_target: &pcie2 {
 	status = "okay";
 };
 
+&pcie1 {
```

Comment:
The commit message for this looks to be largely vestigial.

Is there any reason this is CM5-specific? Doesn't it belong on bcm2712-ds.dtsi?

Reply:
It's not specific to either model (and inadvisable to change - needs to match up with firmware setup of the other bus priorities). So it can go in the base downstream dtb now.